cpp/src_prims/linalg/gemm.cuh

```diff
@@ -22,8 +22,8 @@
 #include <raft/handle.hpp>
 #include "cutlass_wrappers.cuh"
 
-namespace raft {
-namespace linalg {
+namespace MLCommon {
```

Comment:
Let's add a comment that CUTLASS-based gemm is staying in this namespace a bit so RAFT doesn't have to take the CUTLASS dependency.